User-Guide.md

```diff
@@ -0,0 +1,171 @@
+# Introduction
```

Comment:
A TOC would be bodacious dude! :guitar: :smile:

Reply:
:-) how predictable you are ;-) yeah, once we have settled on the order of the items then I'll toc it up before first non-RFC PR :-)

Reply:
Yeah, this is "chicken and egg" stuff, but even now it's difficult to see the structure without a TOC. Sigh... would be sooo easy with rst... ;)